Scripts/Interfaces/IUsesHandedRayOrigin.cs

```diff
@@ -0,0 +1,15 @@
+﻿using UnityEngine.Experimental.EditorVR.Tools;
+
+namespace UnityEngine.Experimental.EditorVR
+{
+	/// <summary>
+	/// Adds Node information to IUsesRayOrigin to determine which hand the tool is attached to
+	/// </summary>
+	public interface IUsesHandedRayOrigin : IUsesRayOrigin
```

Comment:
I suppose we could still have IUsesHandedRayOrigin, but it should combine IUsesRayOrigin and IUsesNode.